docs/monads.md

```diff
@@ -306,7 +306,7 @@ use.
 
 ## Potentially impure code
 
-This is the code that can be instantiated to be either pure or impure:
+The following code can be instantiated to be either pure or impure:
```

Comment:
The original meaning was to clarify what the term "potentially impure code" means. Potentially impure code is the code that can be instantiated to be either pure or impure. I guess ending this sentence with a colon was confusing.